src/app/test-record/seatbelt-installation-check/seatbelt-installation-check.component.html

```diff
@@ -0,0 +1,32 @@
+<dl *ngIf="!editState" class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Carried out during test
+    </dt>
+    <dd class="govuk-summary-list__value" id="test-seatbeltInstallationCheckDate">
+      {{ testType.seatbeltInstallationCheckDate | DefaultNullOrEmpty }}
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Number of seatbelts fitted
+    </dt>
+    <dd class="govuk-summary-list__value" id="test-numberOfSeatbeltsFitted">
+      {{ testType.numberOfSeatbeltsFitted | DefaultNullOrEmpty }}
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Most recent installation check
+    </dt>
+    <dd class="govuk-summary-list__value" id="test-lastSeatbeltInstallationCheckDate">
+      {{ testType.lastSeatbeltInstallationCheckDate | DefaultNullOrEmpty }}
+    </dd>
+  </div>
+</dl>
+
+<ng-container *ngIf="editState && !hidden">
```

Comment:
you can update the name `hidden` it is reserved word. You can use `should..., has.., Is..... for that hasBelt` or something along that line